provision/initramfs/capabilities/transport-http/wwgetfiles

```diff
@@ -108,14 +104,10 @@ while true; do
                 else
                     echo "ERROR: Could not download $name"
                     rm -f $NEWROOT/$path.ww $NEWROOT/$path.ww$timestamp
-                    # Remove lock file
-                    rm -f ${LOCKF}
                     exit 2
```

Comment:
Add in a `exec 9>&-` prior to the exit here? Just to be explicit on releasing the lock ...

Reply:
Why? On exit the open filehandles (and locks on them) are released anyways.

I'd actually prefer the block/subshell-syntax suggested in `man flock`, but I'm not sure if all variants of /bin/sh would accept it.

Reply:
Call it paranoia mostly. It _should_ be released yes...

Reply:
If a terminated process can hold filehandles and locks then something is very wrong with the system.

Reply:
How about `flock -u -n 9` and log explicitly if it fails. On the very odd instance that this happens, we'd at least have a log message to debug with then. Letting the script just exit means we loose the chance to log it.

Reply:
>         -u, --unlock
>              Drop a lock.  *This is usually not required, since a lock is automatically  dropped when*
>              *the  file  is  closed.*  However, it may be required in special cases, for example if the
>              enclosed command group may have forked a background process which should not be  holding
>              the lock.

We do not fork background processes (and even if we did, they *should* hold the lock).

Reply:
Fair enough.